index.html

```diff
@@ -58,7 +59,7 @@
             </div>
             <div class="port-area hide">
                 <div class="port-pic port2"></div>
-                <div class="port-text"><p class="port-text-font">Coming Soon</p></div>
+                <div class="port-text"><p class="port-text-font brand-colour">Coming Soon</p></div>
```

Comment:
I'm pretty sure you could move all the styles from 'port-text-font' to 'port-text' and save using the class and a few lines of CSS? Even if you *have* to have it as a different set of styles, why not target these instead using '.port-text>p' and save needing the class